.github/workflows/on-release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release binaries
+on:
+  push: # TODO: remove?
```

Comment:
unfortunately this workflow won't run before this PR is merged

Reply:
Let's merge it :)